tests/test_georaster.py

```diff
@@ -584,6 +634,27 @@ def test_reproject(self, example: str) -> None:
 
             plt.show()
 
+        # - Check that if mask is modified afterwards, it is taken into account during reproject - #
+        # Create a raster with (additional) random gaps
+        r_gaps = r.copy()
+        data_1d = r_gaps.data.reshape(-1)
+        valids = np.where(~np.ma.atleast_1d(data_1d).mask)[0]
+        assert len(valids) > 0  # sanity check
+        nsamples = 200
+        data_1d[np.random.choice(valids, nsamples, replace=False)] = np.ma.masked
```

Comment:
Same comment here on fixing a random state. Could even use the `subsample_raster` function of GeoUtils.

Reply:
I forgot about `subsample_raster` ! 🙃 I will use that functionality instead then. I have basically just replicated what `subsample_raster` is doing...

Reply:
Use `subsample_raster` since 0781e4b.